cmd/dvara/main.go

```diff
@@ -35,6 +35,7 @@ func Main() error {
 	serverClosePoolSize := flag.Uint("server_close_pool_size", 1, "number of goroutines that will handle closing server connections.")
 	serverIdleTimeout := flag.Duration("server_idle_timeout", 60*time.Minute, "duration after which a server connection will be considered idle")
 	username := flag.String("username", "", "mongo db username")
+	verbose := flag.Bool("verbose", false, "Be really verbose")
```

Comment:
This is just a note for the future, no change needed now:
If I were doing this I'd put all the flags in a `var()` block outside of functions and would call `flag.parse()` from 'real' `main`.

```go
var(
    verbose = flag.Bool(.......)
    otherflag = flag.String(.......)
)

func main() {
    flag.Parse()
    if err := Main(); err != nil {
        os.Exit(1)
    }
}
```